src/main/java/uk/gov/beis/els/api/openapi/OpenApiPropertyCustomiser.java

```diff
@@ -175,4 +176,25 @@ private void processEnumValues(List<Annotation> annotations, Schema schema) {
           }
         });
   }
+
+  private void processExamples(Schema schema) {
+    if (schema.getType().equals("integer")) { // process integers
+      if (schema.getDescription().contains("pixels")) {
+        schema.setExample("100"); // set pixels to something the user can actually see (i.e. bigger than 1 pixel)
```

Comment:
@jamesbarnett91 I did think about creating a specific annotation for pixel fields and QR code fields (processed below) and using that to determine the example property, but that might be overkill. For now it's based on the description. Thoughts? Could also base it on field name? Either could be inconsistent.

Reply:
Done